ground/gcs/src/plugins/uavobjects/uavobjectfield.cpp

```diff
@@ -616,7 +616,11 @@ QString UAVObjectField::toString()
     sout.append ( QString("%1: [ ").arg(name) );
     for (unsigned int n = 0; n < numElements; ++n)
     {
-        sout.append( QString("%1 ").arg(getDouble(n)) );
+        if (type == ENUM) {
+            sout.append( QString("%1 ").arg(getValue(n).toString()) );
+        } else {
+            sout.append( QString("%1 ").arg(getDouble(n)) );
+        }
```

Comment:
Can't you just do QVariant::toString for everything? http://doc.qt.io/qt-5/qvariant.html#toString